### DISCUSSION TITLE
quick start is not working or not binding to localhost:8080 - what log files should I look at?

### DISCUSSION THREAD

**Discussion by matthew-broadbent** [DefectDojo/django-DefectDojo#4732] [Q&A]
I have tried following the quick start method in order to get the application up and running. 

On `docker-compose up`, none of the logs seemed to indicate that any errors occurred. The terminal seemed to indicate the server had been started, but there was no connection.

![image](https://user-images.githubusercontent.com/53876387/123811799-93822600-d8c1-11eb-8fea-a1096fbc3235.png)
![image](https://user-images.githubusercontent.com/53876387/123811860-a09f1500-d8c1-11eb-904f-c623acd8098f.png)

**Comment by damiencarol** [comment]
According to our docker-compose file dedicated to quick start, you should take a look a the `nginx` container.
It is configured to bind on port `8080`.
The common mistake is to have already a program that use this port.

**Comment by valentijnscholten** [comment]
You only provide a small part of the logs, so it's hard to debug, but it looks more like a line ending problem. Are you using Windows?

**Reply by valentijnscholten** [reply]
The log output seems to indicate a line ending issue, hence my question about windows. You should make sure line endings are LF, i.e. set core.autocrlf=false in git before checking out the repo
